find.py

```diff
@@ -0,0 +1,11 @@
+from pymongo import MongoClient
+
+client = MongoClient( '127.0.0.1', 27017)
+db = client.course_shop
+contents = db.contents
+
+contents.find(limit=3)
+contents.find({'price': {"$lte" :30 }})
+contents.find({"categories": "pants",'price': {"$gte" :80 , "$lte": 150}})
+contents.find({"$or" :[{"categories": "ties"}, {"categories": "suits"}]})
```

Comment:
`$or` joins two queries, there's a better operator for checking if a value is "in" a list of values 😄